decode.d.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Is there a way to keep typings in e.g. `typings` folder?

It'll be great to not mix them with regular modules (same as e.g. we keep tests in `test` folder) ?

Reply:
I tried doing that, by specifying "typings" flag in package.json but seems it's not working also we have seprate file based import as well

Reply:
> also we have separate file based import as well

What exactly do you mean by that?

Reply:
means you can use same function multiple ways

```jsx
import qs from "querystring/encode"
import encode from "querystring/encode"
```

Reply:
Indeed, there's seems no way to keep then in dedicated dir. It's a pity, anyway let's go with this